pom.xml

```diff
@@ -106,12 +128,45 @@
 						</goals>
 						<phase>process-resources</phase>
 						<configuration>
-							<propertyFile>db.properties</propertyFile>
+							<propertyFile>liquibase.properties</propertyFile>
 							<changeLogFile>db\migration\dbchangelog.xml</changeLogFile>
 						</configuration>
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<version>${maven-shade-plugin-version}</version>
+				<configuration>
+					<createDependencyReducedPom>false</createDependencyReducedPom>
+				</configuration>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<transformers>
+								<transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+									<manifestEntries>
+										<Main-Class>com.mturko.bestshop.BestShopApplication</Main-Class>
+									</manifestEntries>
+								</transformer>
+								<transformer
+										implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+									<resource>META-INF/spring.handlers</resource>
```

Comment:
I don't see this file in the project. Is it generated by the plugin?

Reply:
it is included inside the jar file itself and is (I believe) generated by the plugin